soda/contracts/docs/README.md

```diff
@@ -0,0 +1,81 @@
+# Implementing data contracts
+
+Data Contracts are the best tool to prevent data issues from occurring. Data contracts
+drives data producers to take ownership of the contract and the data.  A contract contains
+information like the owner, schema, arrival times, data quality checks and is used for the
+following use cases.
+
+1) Source of information for data discovery tools
+2) Verify the contract on new data and alert the producer if not compliant
+3) Verify the contract on pipeline code changes and alert the producer if not compliant
+
+Use contracts if you want to prevent data issues. It's a tool used by the producer, which
+are the technical engineering team responsible for the pipeline component producing the data.
+
+It's a technical component used for encapsulation in the software engineering sense. A data
```

Comment:
encapsulation of what? "encapsulation of producing data" perhaps?

Reply:
Updated the description to:

> Data contracts are a solution used for applying encapsulation in the software engineering sense to data pipeline components. A data contract specifies the interface of the software component producing the data. 

Better?